src/index/tests/index_tests.rs

```diff
@@ -1681,14 +1681,3 @@ fn string_type_alias_without_size_is_indexed() {
     let dt = index.find_effective_type_by_name(my_alias).unwrap();
     assert_eq!("WSTRING", dt.get_name());
 }
-
-#[test]
-fn init_function_is_indexed_unconditionally() {
```

Comment:
Init functions are now indexed "organically" after the lowering stage, so the symbol is no longer present at the original indexing stage and this test no longer makes sense.